src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Tabs/basic.html.twig

```diff
@@ -28,6 +28,29 @@
 <div role="tabpanel" class="form-contenttab tab-pane container-fluid active" id="basic-tab">
   <div class="row">
     <div class="col-md-9 left-column">
+      <div 
+        id="product-images-container"
+        data-translations="{{
+          {
+            'window.selectAll': 'Select all'|trans({}, 'Admin.Catalog.Feature'),
+            'window.unselectAll': 'Unselect all'|trans({}, 'Admin.Catalog.Feature'),
+            'window.selectedFiles': 'selected file(s)'|trans({}, 'Admin.Catalog.Feature'),
+            'window.useAsCover': 'Use as cover image'|trans({}, 'Admin.Catalog.Feature'),
+            'window.saveImage': 'Save image settings'|trans({}, 'Admin.Catalog.Feature'),
+            "window.delete": "Delete selection"|trans({}, 'Admin.Actions'),
+            "window.close": "Close window"|trans({}, 'Admin.Actions'),
+            "window.zoom": "Zoom on selection"|trans({}, 'Admin.Actions'),
+            "ToLargeFile": "The file is too large. Maximum size allowed is: [1] MB. The file you are trying to upload is [2] MB."|trans({}, 'Admin.Notifications.Error')|replace({ '[1]': '{{maxFilesize}}', '[2]': '{{filesize}}' }),
```

Comment:
since the previous wording keys are formatted this way `window.zoom` It would make sense to keep the same format for the last ones (Besides To is missing an O)

Reply:
But these keys are old ones from product page :/